### DISCUSSION TITLE
Analyzer duplicating check on CSLA0017 Business rule missing Add method and not checking for add method on results

### DISCUSSION THREAD

**Discussion by Brannos1970** [MarimerLLC/csla#4067] [General]
In all of my rules I am having to disable CSLA0017 twice. I checked my dependencies and it looked correct. Is there something else I should be looking at?

Also, I noticed that the analyzer does not check if the context.Results collection Add method was used? Not a big deal, as I am ignoring it, but thought I might mention it as it would filter out false positives.

**Comment by rockfordlhotka** [comment]
I don't know.

@kant2002 has been working on the analyzers recently, and so might have some insight?
